source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/ChoiceSetInputRenderer.java

```diff
@@ -46,6 +47,28 @@
 import java.util.List;
 import java.util.Vector;
 
+class ItemSelectedListener implements AdapterView.OnItemSelectedListener
```

Comment:
>class ItemSelectedListener implements AdapterView.OnItemSelectedListene [](start = 0, length = 71)

Could you add comments to this class explaining why it is needed and what it is doing for future reference? Thanks!

Reply:
that class was not needed XD